[2025-01-07T00:02:22.920+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-07T00:01:59.393323+00:00 [queued]>
[2025-01-07T00:02:23.289+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-07T00:01:59.393323+00:00 [queued]>
[2025-01-07T00:02:23.968+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-01-07T00:02:25.910+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_posting_f> on 2025-01-07 00:01:59.393323+00:00
[2025-01-07T00:02:25.936+0000] {standard_task_runner.py:60} INFO - Started process 1069 to run task
[2025-01-07T00:02:25.948+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_insert_log', 'ft_posting_f', 'manual__2025-01-07T00:01:59.393323+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/dag_insert_log.py', '--cfg-path', '/tmp/tmpmb4bdapp']
[2025-01-07T00:02:25.965+0000] {standard_task_runner.py:88} INFO - Job 78: Subtask ft_posting_f
[2025-01-07T00:02:26.246+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-07T00:01:59.393323+00:00 [running]> on host 22295d32c47d
[2025-01-07T00:02:26.719+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='saida' AIRFLOW_CTX_DAG_ID='dag_insert_log' AIRFLOW_CTX_TASK_ID='ft_posting_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T00:01:59.393323+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T00:01:59.393323+00:00'
[2025-01-07T00:02:27.034+0000] {base.py:83} INFO - Using connection ID '***-db' for task execution.
[2025-01-07T00:02:30.590+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-07T00:02:30.604+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_insert_log, task_id=ft_posting_f, execution_date=20250107T000159, start_date=20250107T000222, end_date=20250107T000230
[2025-01-07T00:02:30.684+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-07T00:02:30.733+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
